atk/src/lib.rs

```diff
@@ -32,6 +32,7 @@ mod rt;
 #[cfg_attr(feature = "cargo-clippy", allow(let_and_return))]
 #[cfg_attr(feature = "cargo-clippy", allow(many_single_char_names))]
 #[cfg_attr(feature = "cargo-clippy", allow(wrong_self_convention))]
+#[allow(unused_imports)]
```

Comment:
At some point it'd be nice if `gir` imports were fixed. :)

Reply:
Indeed, but that's quite non-trivial to get right :)